modules/KIWIImage.pm

```diff
@@ -4125,9 +4125,17 @@ sub extractLinux {
     my $xml       = $this->{xml};
     my $vconf     = $xml -> getVMachineConfig();
     my $xendomain;
-    my $kernname = "vmlinuz";
-    if (-f "$imageTree/boot/vmlinux") {
-        $kernname = "vmlinux";
+    my $kernname;
+    foreach (
+        "image",   # s390x
+        "vmlinux", # ppc*
+        "Image",   # aarch64
+        "uImage",  # armv* (legacy)
+        "zImage",  # armv*
+        "vmlinuz", # x86*
+        ) {
```

Comment:
Hmm, actually this should be handled properly in KIWIConfig.sh here:
https://github.com/openSUSE/kiwi/blob/master/modules/KIWIConfig.sh#L1548

The cases you have added here are handled there too, we were also able to build for arm in the past
Thus I'm confused